script.js

```diff
@@ -1,12 +1,105 @@
 //You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+let arrayOfEpisodes = getAllEpisodes();
+let mainEl = document.querySelector("main");
+let searchBar = document.querySelector("#searchInput");
+let displayNumberOfEpisodes = document.querySelector("#displayEpisodesText");
+let selectEl = document.querySelector("#selectMenu");
+
+//search bar
+searchBar.addEventListener("keyup", (e) => {
```

Comment:
All good! There is also another event which is called "input". You can use that and see how it works! :)